responses/__init__.py

```diff
@@ -418,6 +418,22 @@ def get_response(self, request):
             preload_content=False,
         )
 
+    def __str__(self):
+        return str(
+            json_module.dumps(
+                {
+                    "content_type": self.content_type,
+                    "headers": self.headers,
+                    "status": self.status,
+                    "url": self.url,
+                },
+                sort_keys=True,
+            )
+        )
+
+    def __repr__(self):
```

Comment:
repr is typically intended to output something that could be put back into a repl to create the object once again. I think we either shouldn't implement it or implement it in a way that it could be evaled again. This [article](https://www.journaldev.com/22460/python-str-repr-functions#what8217s-the-difference-between--str-and--repr) has an example of how repr and str could differ.

Reply:
Totally agree, in this case, I went for the second option mentioned on the [docs](https://docs.python.org/3/reference/datamodel.html#object.__repr__):

_If at all possible, this should look like a valid Python expression that could be used to recreate an object with the same value (given an appropriate environment). If this is not possible, a string of the form <...some useful description...> should be returned._

I believe that the Response's URL is, for most of the cases, the most used attribute and it could be easily checked when we call, for instance, `Responses.mock.registered()` or just displaying a Response instance in a debugging session, as in:
```
> response = Response(method=responses.GET, url="http://example.com/")
> response
<responses.Response: http://example.com/>
```

What do you think?

Reply:
I agree that the URL is the most important attribute followed by the HTTP method.